### PR TITLE
Color-Game-using-Tkinter

### LINKED ISSUE #739: Color Game using Tkinter
### Aim
To make a tricky and interesting Color game using Tkinter with a GUI. 

### Details
In this game, the player has to enter the color of the word that appears on the screen. he game uses a variety of colors such as Red, Blue, Green, Pink, Black, Yellow, Orange, White, Purple, and Brown. These colors are interchangeably displayed to trick the player’s mind. The graphical user interface (GUI) will display the name of different colors in different colors. 

@avinashkranjan I am a GSSoC'21 participant and I think this is an interesting python script of GUI domain. Kindly allow me to work on this issue and contribute to this project!

### PR DESCRIPTION
**Pull request by PriyanshuChatterjee**
# Description

I have made a game using the Tkinter module in Python. This refers to issue number #739  which was assigned to me. No dependencies are required for the change. 

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@PriyanshuChatterjee once the timer is up, add an option to restart the game and exit the game

**Comment by PriyanshuChatterjee** [comment]
Okay @santushtisharma10 adding that

**Comment by kaustubhgupta** [comment]
@PriyanshuChatterjee when you will make the changes?

**Comment by PriyanshuChatterjee** [comment]
@santushtisharma10 @kaustubhgupta I have added the feature that will enable players to restart the game after one session, or at any point of time in the play.

**Comment by PriyanshuChatterjee** [comment]
@kaustubhgupta Sorry for the delay

File: Color-Game-using-Tkinter/script.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                              + str(timeleft))
+
+        # calling the function again after 1 second
+        timeLabel.after(1000, countdown)
+
+# The Game GUI starting Function
+def vp_start_gui():
+    global root
+    # Score of the player, initially zero
+    global score
```
Comment (kaustubhgupta) [inline Color-Game-using-Tkinter/script.py:71]: Define the score variable outside the functions, that's why test failed

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by santushtisharma10** [comment]
@PriyanshuChatterjee your code is almost same as this [code](https://www.geeksforgeeks.org/color-game-python/)

**Comment by PriyanshuChatterjee** [comment]
@santushtisharma10 Yes its is similar, I took the idea from there. If it is violating the rules, then I'm apologizing and will close this issue.

File: Color-Game-using-Tkinter/script.py (hunk 1 above)
Comment (PriyanshuChatterjee) [reply Color-Game-using-Tkinter/script.py:71]: I cannot implement it by defining them outside the functions.

**Comment by PriyanshuChatterjee** [comment]
@kaustubhgupta @santushtisharma10 extremely apologizing for any mis conduct which I have have committed. Doing open source for the first time, I messed up. I look forward to make other new issues in this repository and will not further repeat my mistake. With that said I'm closing this issue.

**Pull request closed without merging**